ci/test.sh

```diff
@@ -26,6 +26,31 @@ GTEST_ARGS="--gtest_output=xml:${CUGRAPH_ROOT}/test-results/"
 DOWNLOAD_MODE=""
 EXITCODE=0
 
+PRHTTP=https://api.github.com/repos/rapidsai/cugraph/pulls/${PR_ID}/files
+fnames=`curl -sb -X GET -H "Accept: application/vnd.github.v3+json" -H "Authorization: token $GHTK"  $PRHTTP | python3 -c "import sys, json; print([labels['filename'] for labels in json.load(sys.stdin)])"`
+cpp_changed="false" python_changed="false" nb_changed="false" doc_changed="false" 
+for fname in ${fnames[@]}
+do
+   if [[ "$fname" == *"cpp/cmake/"* || "$fname" == *"cpp/CMakeLists.txt"* || "$fname" == *"cpp/src/"* || "$fname" == *"cpp/include/"* || "$fname" == *"cpp/tests/"* || "$fname" == *"cpp/libcugraph_etl/"* || "$fname" == *"cpp/scripts/"* ]]; then
+      cpp_changed="true" python_changed="true" nb_changed="true" doc_changed="true"
+   fi
+   if [[ "$fname" == *"python/"* ]]; then
+      python_changed="true" nb_changed="true" doc_changed="true"
+   fi
+   if [[ "$fname" == *"docs/"* ]]; then
+      doc_changed="true"
+   fi
+   if [[ "$fname" == *"notebooks/"* ]]; then
+      nb_changed="true"
+   fi
+done
+
+if [[ "$nb_changed" == "false" ]]; then
+   export NB_CHANGED="FALSE"
+else
+   export NB_CHANGED="TRUE"
+fi
+
```

Comment:
Somewhere around here might be a good place to print a summary of what's going to be run, what's going to be skipped, and (most importantly) why, since it might be confusing to devs who are expecting certain tests to run which end up getting skipped.

Reply:
Fair ask, will do